spec/requests/api/users_spec.rb

```diff
@@ -110,17 +110,22 @@
     end
 
     it 'should return 400 error if name not given' do
-      post api('/users', admin), email: 'test@example.com', password: 'pass1234'
+      post api('/users', admin), attributes_for(:user).delete(:name)
       expect(response.status).to eq(400)
     end
 
     it 'should return 400 error if password not given' do
-      post api('/users', admin), email: 'test@example.com', name: 'test'
+      post api('/users', admin), attributes_for(:user).delete(:password)
       expect(response.status).to eq(400)
     end
 
     it "should return 400 error if email not given" do
-      post api('/users', admin), password: 'pass1234', name: 'test'
+      post api('/users', admin), attributes_for(:user).delete(:email)
+      expect(response.status).to eq(400)
+    end
+
+    it "should return 400 error if username not given" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.